tests/test_cli/test_launch_end_to_end.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This test module contains the tests for the `aea launch` sub-command."""
+import json
+import os
+import sys
+import uuid
+
+from aea.test_tools.test_cases import AEATestCaseMany
+
+from tests.common.pexpect_popen import PexpectWrapper
+
+
+class TestLaunchEndToEnd(AEATestCaseMany):
+    """Perform aea launch end to end test."""
+
+    key = "seller_service"
+    value = "generic_service"
```

Comment:
This can clash with other tests, try use something specific to the test

Reply:
it's overrided in test method